### ISSUE TITLE
GRSciColl - identify Natural History collections and institutions

### ISSUE THREAD

**Issue by ManonGros** [gbif/registry#436]
GRSciColl contains collections that aren't related to Natural History. See this example: https://registry.gbif.org/institution/c24bd5dc-8cd3-4364-8b5c-84bc41dccdd1
Although we want to keep those entries, it would be good to be able to filter for Natural History collections and institutions (especially in the context of Natural History Collection portal(s)).

I don’t think we can actually use the `contentType` and `Discipline` fields. One reason is that some archeological or other collections might be relevant for biodiversity (fur coat or feather crown, etc). We might have to rely on manual tagging.

Since I would probably do the tagging, I would like to do the least amount of maintenance possible. It would be better to have all new institutions and collections assumed as NHC by default and having to only find and handle the entries to exclude.

I am not sure how this could work technically as machineTags cannot exclude records in the search. Perhaps a default machineTag? @marcos-lg

**Comment by marcos-lg** [comment]
I was just talking to Morten and I think it might be easier to add a new field because if we do it with a machine tag then we'll have to create the machine tag for every new collection/institution created and it can be a bit annoying. This field could be a boolean value that defaults to true or we can create an enumeration if we expect to have more than 2 values. If we use a new field you wouldn't have to set the value manually so it won't be no work for you (only to exclude the non-natural-history ones)

**Comment by MortenHofft** [comment]
If we do this, then we probably need to add the same parameter to suggest, to avoid suggesting collections that isn't relevant

**Comment by MortenHofft** [comment]
In discussion with @ahahn-gbif we decided to implement this, as we want to to exclude those data from a grscicoll website and we want it to be as simple and flexible as possible for data managers. So instead of trying to infer relevance from other fields, we opted for a simple flag. The flag will default to including everything. With the option for editors to then remove it from a site focusing on biodiversity relevant collections.
